Src/App/Router.php

```diff
@@ -8,30 +8,25 @@ class Router
     /**
      * lead to the controller
      * @param $parameters
-     * @return mixed
+     * @return array
      * @throws \Exception
      */
     public function route($parameters)
     {
-
+        $controllerClassName = ucfirst($parameters['controller']) . 'Controller';
         if (isset($parameters['controller']) && isset($parameters['method'])) {
-            $controllerClassName = ucfirst($parameters['controller']) . 'Controller';
             $controllerName = '\Aot\Controller\\' . $controllerClassName;
             $method = $parameters['method'];
         } else {
             throw new \Exception("method or controller invalid");
         }
         if (class_exists($controllerName)) {
             $controller = new $controllerName();
+            unset($parameters['method'], $parameters['controller']);
+            return $controller->action($parameters, $method);
```

Comment:
How do know `action` exists in the controller?

Reply:
action is hard coded should i control it anyway?